src/main.js

```diff
@@ -30,12 +31,13 @@ export async function run() {
       core.info('Branch matched prefix: ' + branch)
       let statusOK = true
 
-      // Check CI status if required
-      if (mustBeGreen) {
+      // Check CI status or review status if required
+      if (mustBeGreen || mustBeApproved) {
```

Comment:
IMHO this should be an `&&`. Or is it expected/wanted that a review can bypass CI?

Reply:
Later on, these conditions are checked individually. If either config option is set to check them, one cannot "bypass the other". If either check fails, we set a flag in our code that it failed and the branch is not eligible for combination.

Reply:
Aight, make sense :)